src/compiler/crystal/tools/doc/html/css/style.css

```diff
@@ -55,8 +67,38 @@ h1.type-name {
   line-height: 1.2;
   width: 100%;
   border: 0;
-  border-radius: 50px;
   outline: 0;
+  border-radius: 2px;
+  box-shadow: 0px 3px 5px rgba(0,0,0,.25);
+  transition: box-shadow .12s;
+}
+
+#types-list input:focus {
+  box-shadow: 0px 5px 6px rgba(0,0,0,.5);
+}
+
+#types-list input::-webkit-input-placeholder { /* Chrome/Opera/Safari */
```

Comment:
I could remove the support for firefox 18-. There probably aren't many people on that older version anymore (from 2012-2013).